tests/components/tuya/fixtures/cz_vrbpx6h7fsi5mujb.json

```diff
@@ -0,0 +1,225 @@
+{
```

Comment:
New fixture should be added to `__init__.py`

https://github.com/home-assistant/core/blob/270a9a5a98cae57542f39f4b6ff7b05e8eb9b9ba/tests/components/tuya/__init__.py#L68-L70